### PR TITLE
Fix type boolean `MYSQL`.

### LINKED ISSUE #19808: Boolean MySQL migration vs. auto-model-attribute-typecast inconsistency
- `TINYINT(1)` column is created using `$this->boolean()` in migration
- `BIT(1)` column is type-casted to `bool` during model population (`TINYINT(1)` column is type-casted as `integer`)

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ✔️

### CODE REVIEW

File: framework/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ Yii Framework 2 Change Log
 
 2.0.50 under development
 ------------------------
+- Bug #20045: Fix type `boolean` in `MYSQL` (terabytesoftw)
```
Comment (rob006) [inline framework/CHANGELOG.md:6]:
```suggestion
- Bug #20045: Fix type `boolean` in `MySQL` (terabytesoftw)
```

**Review by samdark — APPROVED** [review]

File: framework/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@ Yii Framework 2 Change Log
 
 2.0.50 under development
 ------------------------
+- Bug #20045: Fix type `boolean` in `MySQL` (terabytesoftw)
```
Comment (bizley) [inline framework/CHANGELOG.md:6]: We have this line twice now?

Comment (terabytesoftw) [reply framework/CHANGELOG.md:6]: image

Nope, MSSQL and MySQL,

Comment (bizley) [reply framework/CHANGELOG.md:6]: I'm blind apparently :) sorry

Comment (terabytesoftw) [reply framework/CHANGELOG.md:6]: no problem.

**Review by Arhell — APPROVED** [review]

**Pull request merged**